openedx/core/djangoapps/profile_images/images.py

```diff
@@ -205,7 +205,8 @@ def _update_exif_orientation(exif, orientation):
     the exif orientation, return a new exif with the orientation set.
     """
     exif_dict = piexif.load(exif)
-    exif_dict['0th'][piexif.ImageIFD.Orientation] = orientation
+    if orientation:
+        exif_dict['0th'][piexif.ImageIFD.Orientation] = orientation
```

Comment:
apparently EXIF orientations are between 1-8 or 0 if none is supplied [1]. Could we autodefault this to 0 if none is supplied?

```python
exif_dict['0th'][piexif.ImageIFD.Orientation] = orientation or 0
```

[1] https://docs.imgix.com/apis/url/rotation/or